android/src/main/java/io/bluedot/AppTempoReceiver.java

```diff
@@ -1,19 +1,21 @@
 package io.bluedot;
 
 import android.content.Context;
+
```

Comment:
🤔 Looks like you have a different import code style which is causing a lot of unrelated changes? What IDE are you using?

Reply:
android Studio 4.1.2

Reply:
Ok, cool, we have a code style file here: https://gitlab.com/bluedotio/android/dev_resources_android/-/blob/master/Bluedot.xml you can dl and import, which should reduce the amount of import moving noise between MRs, amongst other benefits. Let me know if any issues.

Reply:
I applied the scheme and reoptimised imports. Please check again.